snorkeling_helper/candidates_helper/label_candidates_modules.py

```diff
@@ -0,0 +1,26 @@
+def char_to_word(row):
```

Comment:
Looks like you also have this defined in your notebook. Maybe delete the one in the notebook?